deploy/webhook/member-operator-webhook.yaml

```diff
@@ -275,6 +275,30 @@ objects:
       namespaceSelector:
         matchLabels:
           toolchain.dev.openshift.com/provider: codeready-toolchain
+    - name: users.virtualmachines.validating.webhook.sandbox
```

Comment:
could you please add a comment explaining what the webhook does? as we have for the other ones above.

Reply:
Good point, thanks, I missed that. Done c3a6cebff8433dc696d08f0dc82c7671e54c953a